grpc/server.go

```diff
@@ -348,3 +348,17 @@ func (s *Server) History(request *pb.HistoryRequest, stream pb.Frames_HistorySer
 
 	return apiError
 }
+
+// Exec executes a command
+func (s *Server) Version(ctx context.Context, req *pb.VersionRequest) (*pb.VersionResponse, error) {
+	request := frames.VersionRequest{}
+
+	version, err := s.api.Version(&request)
```

Comment:
again, no need to pass anything to the `s.api.Version`
Or alternatively just return `s.version`